source/azure_iot_hub_client.c

```diff
@@ -109,7 +109,7 @@ static void prvMQTTProcessSuback( AzureIoTHubClient_t * pxAzureIoTHubClient,
 
         if( pxContext->_internal.usMqttSubPacketID == usPacketId )
         {
-            /* TODO: inspect packet to see is ack was successful*/
+            /* We assume success since IoT Hub would disconnect if there was a problem subscribing. */
```

Comment:
Can we also add same comment in provisioning client 

https://github.com/Azure/azure-iot-middleware-freertos/blob/33db1d22451dec6b07302de55c9ba70c711f0f30/source/azure_iot_provisioning_client.c#L566

Reply:
added